common/transform/src/main/java/ai/h2o/mojos/deploy/common/transform/Utils.java

```diff
@@ -42,4 +47,27 @@ public static String sanitizeBoolean(String value, DataField.DataTypeEnum dataTy
     }
     return value;
   }
+
+  /**
+   * Check if prediction interval is enabled or not.
+   */
+  public static boolean isPredictionIntervalEnabled() {
+    return isPredictionIntervalEnabledFromEnv() || isPredictionIntervalEnabledFromProperty();
+  }
```

Comment:
My two cents is the methods here are overkill. Consider placing all of the logic into a single function and keep it close to the place it needs to be used (not in `Utis.java`).

Reply:
make sense